### PR TITLE
7699 fix req to ens release - RELEASE

### LINKED ISSUE #7699: can't seem to add ens contact
# Bug Report

## Steps to reproduce

1. go to settings -> messaging -> contacts -> send contact request using key
2. write ens username (a checkmark should appear)
3. write message and send request

result: receiving user does not get a request

### PR DESCRIPTION
**Pull request by endulab**
Fix #7699 

### What does the PR do

Send request to correct chat key for ens account.
Improve pasting chat key - the correctness is automatically checked w/o the need to click on input.

### Affected areas

Chat / Send Contact Request

### Screenshot of functionality (including design for comparison)

image

### CODE REVIEW

File: ui/app/AppLayouts/Profile/popups/SendContactRequestModal.qml [hunk 1]
```diff
@@ -40,6 +41,23 @@ StatusModal {
         property var lookupContact: Backpressure.debounce(root, 400, function (value) {
             root.contactsStore.resolveENS(value)
         })
+
+        function textChanged(text) {
```
Comment (MishkaRogachev) [inline ui/app/AppLayouts/Profile/popups/SendContactRequestModal.qml:45]: Mb `resolvePublicKey(chatKey)`  is better name for this method?

**Comment by caybro** [comment]
@endulab just a reminder, this needs to land in master too

**Comment by caybro** [comment]
> @endulab just a reminder, this needs to land in master too

Replying to myself: I guess it'd be much better if @iurimatias cherry-picked these fixes to the 0.7 branch manually

**Comment by endulab** [comment]
> > @endulab just a reminder, this needs to land in master too
> 
> Replying to myself: I guess it'd be much better if @iurimatias cherry-picked these fixes to the 0.7 branch manually

There is master branch too: https://github.com/status-im/status-desktop/pull/7716

File: ui/app/AppLayouts/Profile/popups/SendContactRequestModal.qml (hunk 1 above)
Comment (endulab) [reply ui/app/AppLayouts/Profile/popups/SendContactRequestModal.qml:45]: This function does not resolve public key. It does a set of operations when text is changed (manually or pasted) , eg. sets some properties and also runs lookupContact which returns resolved name.

**Review by anastasiyaig — APPROVED** [review]

**Pull request closed without merging**